### PR TITLE
Adding ELI5 video to the home page

### PR DESCRIPTION
**Pull request by dmitryvinn**
## Summary
Here at Meta Open Source, we've been creating short-form videos to explain Meta OSS projects in the simplest terms possible. We started embedding these videos into project pages as you can see in [Docusaurus home page](https://docusaurus.io/), [Jest](https://jestjs.io/), [Litho](https://fblitho.com/), [Hydra](https://hydra.cc/), etc.

Although Ent has moved to Linux foundation, I wanted to propose adding this video to Ent as well, as it's been getting decent views and might benefit the community.

## Test plan
image

### CODE REVIEW

**Comment by dmitryvinn** [comment]
@a8m what do you think?

**Review by a8m — APPROVED** [review]
Thanks for adding this @dmitryvinn. I’d love to add more like this one to our homepage.

I want to play a bit with its design and see if I can find a better composition for it, and then merge it! 🙏🏻
